src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Hello</title>
+  </head>
+  <body>
+Hello, world!
```

Comment:
don't forget about whitespaces
```suggestion
    Hello, world!
```